propagator/ottrace/lib/opentelemetry/propagator/ottrace/text_map_extractor.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+module OpenTelemetry
+  module Propagator
+    module OTTrace
+      class TextMapExtractor
+        PADDING = '0' * 16
+        VALID_TRACE_ID_REGEX = /^[0-9a-f]{32}$/.freeze
+        VALID_SPAN_ID_REGEX = /^[0-9a-f]{16}$/.freeze
+
+        # Returns a new TextMapExtractor that extracts OTTrace context using the
+        # specified getter
+        #
+        # @param [optional Getter] default_getter The default getter used to read
+        #   headers from a carrier during extract. Defaults to a
+        #   {OpenTelemetry::Context:Propagation::TextMapGetter} instance.
+        # @return [TextMapExtractor]
+        def initialize(
+          baggage_manager:,
```

Comment:
Is it necessary to inject the `baggage_manager`, or we can just assume it's `OpenTelemetry.baggage`?

Reply:
@fbogsany made a similar observation here: https://github.com/open-telemetry/opentelemetry-ruby/pull/630#discussion_r588774083

Are you both referring to the fact that it does not have a default value or  do you mean that the `inject` method should reference the `OpenTelemetry.baggage` (service locator) instead of using constructor injection?